.github/workflows/CI.yml

```diff
@@ -59,14 +59,51 @@ jobs:
         run: yarn circuits circuit:setup:plonk
 
       - name: Relayer Tests
-        run: yarn relayer test
+        run: |
+          yarn relayer test
+          echo "TESTNET_RELAYER_ACCOUNT_ID=$TESTNET_RELAYER_ACCOUNT_ID" >> $GITHUB_OUTPUT
+          echo "TESTNET_RELAYER_PRIVATE_KEY=$TESTNET_RELAYER_PRIVATE_KEY" >> $GITHUB_OUTPUT
+          echo "TESTNET_HYC_CONTRACT=$TESTNET_HYC_CONTRACT" >> $GITHUB_OUTPUT
 
       - name: Rust Tests
         run: yarn contracts test:rust
 
       - name: Lib Tests
         run: yarn contracts test:lib
 
+      - name: Building Relayer
+        run: yarn relayer build
+
+      - name: Log Envs
+        run: |
+          echo "Relayer Account Id"
+          echo ${{ env.TESTNET_RELAYER_ACCOUNT_ID }}
+          echo "Relayer Private key"
+          echo ${{ env.TESTNET_RELAYER_PRIVATE_KEY }}
+          echo "Relayer HYC Contract"
+          echo ${{ env.TESTNET_HYC_CONTRACT }}
+
+      - name: Publish
+        uses: cloudflare/wrangler-action@2.0.0
+        with:
+          apiToken: ${{ secrets.CF_API_TOKEN }}
+          command: publish --name dev-relayer
+          workingDirectory: 'packages/relayer'
+          secrets: |
+              RELAYER_FEE
+              NEAR_NETWORK
+              ACCOUNT_ID
+              PRIVATE_KEY
+              HYC_CONTRACT
+              RPC_URL
+        env:
+          RELAYER_FEE: "0"
```

Comment:
Why isn't the fee coming from an env too?

Reply:
I don't like a common env for all packages, but i will change this